### PR TITLE
FizzBuzz

### PR DESCRIPTION
**Pull request by coryjanes**
Added FizzBuzz needed files

### CODE REVIEW

**Review by Tech-Wizz — CHANGES_REQUESTED** [review]
You forgot about the FizzBuzz for 15 since it is a 3 and a 5.  I only got Fizz.

**Comment by Tech-Wizz** [comment]
For got to show evidence of my running the code (+10)
FizzBuzzCory

**Comment by JamesLucas943** [comment]
image

Same thing as Tech-Wizz. The code runs great, but you need to make sure that for numbers that are divisible by both 3 and 5 'FizzBuzz' is printed rather than just 'Fizz'. Everything else looks great.

File: fizzbuzz/python/coryjanes/fizzbuzz.py
Comment (Tech-Wizz) [inline fizzbuzz/python/coryjanes/fizzbuzz.py:1]: Runs correctly now. Pulled down to my machine and it runs. +10

**Review by Tech-Wizz — APPROVED** [review]

**Pull request merged**